Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -200,17 +200,33 @@ def remediation_command_list_request(self, time_range: Dict[str, Any], alert_ids
     def alert_remediate_request(self, alert_id: str):
         self._http_request('PATCH', f'alert/remediation/{alert_id}', resp_type='response')
 
+    def remove_additional_resource_fields(self, input_dict, keys):
```

Comment:
I would avoid recursion. instead lets just delete the fields one by one.

Reply:
These fields could appear several times, and I need to modify the object as I go.